### PR TITLE
test(validators): close coverage gaps in is_*/assert_* paths

### PR DESCRIPTION
**Pull request by bluetoothbot**
## What
Adds edge-case tests that bring `src/dbus_fast/validators.py` from 83% to 100% coverage.

## Why
The `assert_*` helpers and several branches in `is_bus_name_valid` / `is_interface_name_valid` had no direct tests — a regression in those paths (e.g., dropping the `.`-requirement check or breaking the 255-char limit) would have slipped through CI.

## How
Pure test additions in `tests/test_validators.py`, no production code touched:

- Unique bus name (`:` prefix) accepted
- Well-known bus name / interface name without `.` rejected
- 255-char boundary for bus, interface, and member names
- All four `assert_*` helpers return `None` on valid input and raise the matching `Invalid*Error` on invalid input

## Testing
`pytest tests/test_validators.py` — 14 passed, validators.py at 100% statement and branch coverage.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

**Comment by bdraco** [comment]
@bluetoothbot Add -> None: to the return type in the tests on this pr, new commit

**Comment by bdraco** [comment]
@bluetoothbot rr Add -> None: to the return type in the tests on this pr, new commit

**Comment by bdraco** [comment]
@bluetoothbot recreate

**Comment by bdraco** [comment]
@bluetoothbot recreate

File: tests/test_validators.py [hunk 1]
```diff
@@ -85,3 +99,84 @@ def test_member_name_validator():
         assert not is_member_name_valid(member), (
             f'member name should be invalid: "{member}"'
         )
+
+
+def test_bus_name_requires_dot():
```
Comment (bdraco) [inline tests/test_validators.py]: missing -> None:

File: tests/test_validators.py [hunk 2]
```diff
@@ -85,3 +99,84 @@ def test_member_name_validator():
...
+    """A single-element well-known bus name (no '.') is invalid."""
+    assert not is_bus_name_valid("foo")
+
+
+def test_unique_bus_name_is_valid():
```
Comment (bdraco) [inline tests/test_validators.py]: missing -> None:

File: tests/test_validators.py [hunk 3]
```diff
@@ -85,3 +99,84 @@ def test_member_name_validator():
...
+    """Unique bus names start with ':' and bypass element validation."""
+    assert is_bus_name_valid(":1.42")
+    assert is_bus_name_valid(":1.0")
+
+
+def test_bus_name_max_length():
```
Comment (bdraco) [inline tests/test_validators.py]: missing -> None:

File: tests/test_validators.py [hunk 4]
```diff
@@ -85,3 +99,84 @@ def test_member_name_validator():
...
+    """Bus names must not exceed 255 characters."""
+    name_255 = "a." + "b" * 253
+    name_256 = "a." + "b" * 254
+    assert len(name_255) == 255
+    assert len(name_256) == 256
+    assert is_bus_name_valid(name_255)
+    assert not is_bus_name_valid(name_256)
+
+
+def test_interface_name_max_length():
```
Comment (bdraco) [inline tests/test_validators.py]: missing -> None:

File: tests/test_validators.py [hunk 5]
```diff
@@ -85,3 +99,84 @@ def test_member_name_validator():
...
+    """Interface names must not exceed 255 characters."""
+    name_255 = "a." + "b" * 253
+    name_256 = "a." + "b" * 254
+    assert is_interface_name_valid(name_255)
+    assert not is_interface_name_valid(name_256)
+
+
+def test_interface_name_requires_dot():
```
Comment (bdraco) [inline tests/test_validators.py]: missing -> None:

File: tests/test_validators.py [hunk 6]
```diff
@@ -85,3 +99,84 @@ def test_member_name_validator():
...
+    """A single-element name without '.' is not a valid interface name."""
+    assert not is_interface_name_valid("foo")
+
+
+def test_member_name_max_length():
```
Comment (bdraco) [inline tests/test_validators.py]: missing -> None:

File: tests/test_validators.py [hunk 7]
```diff
@@ -85,3 +99,84 @@ def test_member_name_validator():
...
+    """Member names must not exceed 255 characters."""
+    member_255 = "a" * 255
+    member_256 = "a" * 256
+    assert is_member_name_valid(member_255)
+    assert not is_member_name_valid(member_256)
+
+
+def test_assert_bus_name_valid():
```
Comment (bdraco) [inline tests/test_validators.py]: missing -> None:

File: tests/test_validators.py [hunk 8]
```diff
@@ -85,3 +99,84 @@ def test_member_name_validator():
...
+    """assert_bus_name_valid returns None for valid names, raises for invalid."""
+    assert assert_bus_name_valid("foo.bar") is None
+    assert assert_bus_name_valid(":1.42") is None
+    with pytest.raises(InvalidBusNameError):
+        assert_bus_name_valid("5foo.bar")
+    with pytest.raises(InvalidBusNameError):
+        assert_bus_name_valid(".foo")
+
+
+def test_assert_object_path_valid():
```
Comment (bdraco) [inline tests/test_validators.py]: missing -> None:

File: tests/test_validators.py [hunk 9]
```diff
@@ -85,3 +99,84 @@ def test_member_name_validator():
...
+    """assert_object_path_valid returns None for valid paths, raises for invalid."""
+    assert assert_object_path_valid("/") is None
+    assert assert_object_path_valid("/foo/bar") is None
+    with pytest.raises(InvalidObjectPathError):
+        assert_object_path_valid("foo")
+    with pytest.raises(InvalidObjectPathError):
+        assert_object_path_valid("/foo/bar/")
+
+
+def test_assert_interface_name_valid():
```
Comment (bdraco) [inline tests/test_validators.py]: missing -> None:

File: tests/test_validators.py [hunk 10]
```diff
@@ -85,3 +99,84 @@ def test_member_name_validator():
...
+    """assert_interface_name_valid returns None for valid names, raises for invalid."""
+    assert assert_interface_name_valid("foo.bar") is None
+    with pytest.raises(InvalidInterfaceNameError):
+        assert_interface_name_valid("foo")
+    with pytest.raises(InvalidInterfaceNameError):
+        assert_interface_name_valid("5foo.bar")
+
+
+def test_assert_member_name_valid():
```
Comment (bdraco) [inline tests/test_validators.py]: missing -> None:

**Comment by bdraco** [comment]
@bluetoothbot reviewrebase

**Review by bdraco — APPROVED** [review]
